atomix/cluster/src/test/java/io/atomix/raft/impl/PriorityElectionTimerTest.java

```diff
@@ -54,7 +57,9 @@ public void shouldLowerPriorityNodeEventuallyStartsAnElection() {
   @Test
   public void shouldHighPriorityNodeStartElectionFirst() {
     // given
-    final AtomicBoolean highPrioElectionTriggered = new AtomicBoolean();
+    final AtomicBoolean highPrioElectionTriggered = spy(new AtomicBoolean());
```

Comment:
Interesting. I thought you just use `mock(Runnable.class)`. I think it is not necessary to use the AtomicBoolean, but if you prefer it like that, I'm fine.

Reply:
I wanted to wait until it is triggered atleast once. When using AtomicBoolean, I can use it's get method to wait. I found it better than adding timeouts to verify.

Reply:
You could also things like `inOrder.verify(mock, timeout(100)).run();`